samps/uiname.properties

```diff
@@ -12,16 +12,17 @@
 #    (the other line may remain, the UI is per login not client type)
 # 3 copy updated uiname.properties to the same directory where pc2v9.ini is installed
 #
+# administrator1=edu.csus.ecs.pc2.ui.admin.AdministratorView
+# administrator1=edu.csus.ecs.pc2.ui.admin.AdministratorLegacyView
+administrator4=edu.csus.ecs.pc2.ui.admin.ReportsView

```

Comment:
This line, as well as six following lines, have DOS CR/LF line terminators; they should use Unix LF terminators.